src/org/loklak/harvester/strategy/KaizenHarvester.java

```diff
@@ -94,6 +109,17 @@ private void grabInformation(Timeline timeline) {
             if (!place.isEmpty())
                addQuery("near:\"" + message.getPlaceName() + "\" within:" + PLACE_RADIUS + "mi");
         }
+
+        if (query != null && oldestTweetDate != null) {
+            String oldestTweetDateStr = dateToString.format(oldestTweetDate);
+            int startIndex = query.indexOf("until:");
```

Comment:
Why `until:`? Twitter search page limits to 20 tweets if the javascript is not executed. I think `since:` is a better choice.